docs/ensuring-repro/docker/updating-images.md

```diff
@@ -0,0 +1,34 @@
+# Updating Docker images
+
+Over the course of module development, new dependencies and packages may get added or removed, thereby changing the [module's software environment](../managing-software/index.md).
+For example, your environment will change if you add a new R or Python package to perform additional analyses.
+Changing the software environment means that module-specific Docker image also needs to be updated to incorporate those changes.
+
+!!! note
+    These instructions assume that the Data Lab has already activated both of the [GitHub Action (GHA) workflows](../../contributing-to-analyses/analysis-modules/creating-a-module.md#module-workflows) that were created when the module was established, including the [module-testing GHA](../workflows/run-module-gha.md) and the [Docker-building GHA](#STUB_LINK../workflows/docker-build-gha.md).
+
+    In addition, the module-testing GHA should have already been updated to build its runtime environment from the module's Docker image, not from other environment files (e.g. `renv.lock` or `conda-lock.yml`.)
+
+    If these conditions are not met, you do not need to take any specific steps to update Docker images when you update a module's software environment.
+
+
+## Context (header help???)
```

Comment:
> Or just don't have a header at all since this is all intro/background before getting into the steps.

I think this is probably best, since the header felt a little forced to me